app/src/main/java/de/danoeh/antennapod/ui/screen/subscriptions/SubscriptionViewHolder.java

```diff
@@ -60,29 +56,16 @@ public void bind(Feed feed, int columnCount, int counter) {
             count.setVisibility(View.GONE);
         }
 
-        CoverLoader coverLoader = new CoverLoader();
-        boolean textAndImageCombined = feed.isLocalFeed() && feed.getImageUrl() != null
-                && feed.getImageUrl().startsWith(Feed.PREFIX_GENERATIVE_COVER);
-        coverLoader.withUri(feed.getImageUrl());
+        new CoverLoader(coverImage, CoverLoader.fromFeed(feed)).load();
```

Comment:
This looks a bit duplicated because CoverLoader shows up twice. How about moving the builder into ImageModel? Something like this:
```suggestion
        new CoverLoader(coverImage, ImageModel.fromFeed(feed)).load();
```

Reply:
I already thought about that, will do it.